src/workerd/io/worker.h

```diff
@@ -228,12 +233,57 @@ class Worker::Script: public kj::AtomicRefcounted {
   inline bool isModular() const {
     return modular;
   }
+  inline bool isPython() const {
+    return python;
+  }
 
   struct CompiledGlobal {
     jsg::V8Ref<v8::String> name;
     jsg::V8Ref<v8::Value> value;
   };
 
+  struct EsModule {
```

Comment:
Nit: a code comment here briefly explaining these structs would be nice. Nonblocking.

Reply:
Done in f2858bcd48ec761434f37108604e68cf90e523f2